### DISCUSSION TITLE
About Internet Access

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#821] [Ideas]
Hello luc, can ESP 3D have the function of remote internet access

**Comment by luc-github** [comment]
this is done by configuring all ports/ip of your ESP to be redirected by your gateway - it is not related to ESP3D actually.

the router need to forward external IP to local IP, web server port (80) and web socket port (81), for camera esp32 it may be be little bit more tricky but it follow same idea

**Reply by luc-github** [reply]
also be sure you enabled authentication - security is not great on ESP3D because it does not support https yet , so remote acces may not be suggested yet, it may bring a breach in your network IMHO
